_search-plugins/search-pipelines/neural-query-enricher.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Neural query enrich
+nav_order: 12
+has_children: false
+parent: Search processors
+grand_parent: Search pipelines
+---
+
+# Neural query enrich processor
+
+The `neural_query_enricher` search request processor is designed to set a default machine learning (ML) model ID at index or field level for [neural search]({{site.url}}{{site.baseurl}}/search-plugins/neural-search/) queries. To learn more about ML models, see [ML Framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/ml-framework/).
+
+## Request fields
+
+The following table lists all available request fields.
+
+Field | Data type | Description
+:--- | :--- | :---
+`default_model_id` | String | The model ID of the default model for an index. Optional. You must specify at least one of `default_model_id` and `neural_field_default_id`. If both are provided, `neural_field_default_id` takes precedence.
```

Comment:
```suggestion
`default_model_id` | String | The model ID of the default model for an index. Optional. You must specify at least one model ID: `default_model_id` or `neural_field_default_id`. If both are provided, `neural_field_default_id` takes precedence.
```

Reply:
Not sure about this because it's an unfortunate name choice. Even though `default_model_id` is a field, `neural_field_default_id` is technically not one model ID but a map of fields to model IDs... So I think I'll leave as is.